02 Toolbar/src/common/managers/textAreaManager.ts

```diff
@@ -0,0 +1,60 @@
+
+// Based on: http://stackoverflow.com/questions/1064089/inserting-a-text-where-cursor-is-using-javascript-jquery
+class TextAreaManager {
+  insertAtCaret(textArea: HTMLTextAreaElement, caret: string, offsetCursor: number = 0): string {
```

Comment:
We have to think for a better name for this method, now it doesn't update the html element text any more.

Any idea?